cmd/config-v17.go

```diff
@@ -26,17 +26,21 @@ import (
 	"github.com/tidwall/gjson"
 )
 
-// Read Write mutex for safe access to ServerConfig.
-var serverConfigMu sync.RWMutex
-
 // Config version
 const v17 = "17"
 
+var (
+	// serverConfig server config.
+	serverConfig      *serverConfigV17
+	serverConfigMutex sync.RWMutex
```

Comment:
in Go, a general convention that I have seen people follow is - for object named xxxx - if it needs lock then the lock is named as xxxxMu